aiorussound/models.py

```diff
@@ -6,6 +6,18 @@
 from mashumaro.mixins.orjson import DataClassORJSONMixin
 
 
+@dataclass
+class RussoundFavorite:
```

Comment:
Let's keep the class name consistent with the other models (excl. RussoundMessage ofc. which is slated to be renamed in future versions).
```suggestion
class Favorite:
```